src/modules/flow/iio/nodes.c

```diff
@@ -959,6 +959,160 @@ light_open(struct sol_flow_node *node, void *data, const struct sol_flow_node_op
 }
 
 static bool
+intensity_both_create_channels(struct iio_double_data *mdata, int device_id)
+{
+    mdata->iio_base.device = sol_iio_open(device_id, &mdata->iio_base.config);
+    SOL_NULL_CHECK(mdata->iio_base.device, false);
+
+    mdata->channel_val = iio_add_channel(mdata->scale, mdata->offset, "in_intensity0_both", &mdata->iio_base);
```

Comment:
I think that as multiple channels support is already in, you can rebase this patch on top of master and make use of that, instead of this `in_intensity0_both` and `in_intensity_both` trick.